src/controllers/places/modify_content.js

```diff
@@ -25,23 +25,25 @@ module.exports = (req, res) => {
   getLatLng(req.body.address, (err, response) => {
     if (!err) {
       apiBody.location = [response.lat, response.lng];
-      console.log(apiBody);
     }
 
+    const url = req.body._method === 'put' ? `${placesURL}/${req.params.id}` : placesURL;
+
     const reqOptions = {
-      url: placesURL,
-      method: 'POST',
+      url: url,
+      method: req.body._method,
       body: apiBody,
       json: true
     };
-
-    Request(reqOptions, (error, postRes, postResBody) => {
+    Request(reqOptions, (error, apiResponse, apiResponseBody) => {
       if (error) {
-        console.log(error);
         res.send('huge failure');
+      }
+      if ((reqOptions.method === 'put' && apiResponse.statusCode !== 200) || (reqOptions.method === 'post' && apiResponse.statusCode !== 201)) {
+        res.status(404).send(apiResponseBody);
       } else {
-        console.log(postResBody);
-        res.redirect(`/${req.params.lang}/places`);
+        const urlEnd = req.body._method === 'put' ? `place/${req.params.id}` : 'places';
```

Comment:
Not 100% happy with the name of this variable but I can't think of anything else to call it so whatever